go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/mark3labs/mcp-go v0.26.0
 	github.com/spf13/cobra v1.9.1
 	github.com/spf13/pflag v1.0.6
+	gopkg.in/yaml.v3 v3.0.1
```

Comment:
I think this package is archived, can we use sigs.k8s.io/yaml ? that's already included.

Reply:
Good catch. Fixed.